docker_entrypoint.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+chown mastodon:mastodon /mastodon/public/system
```

Comment:
https://github.com/tootsuite/mastodon/issues/3194 is why this part got removed.
By the way, the old entrypoint was also targeting this single directory. So I'm a bit confused.